src/Tools/Source/RunTests/ProcessTestExecutor.cs

```diff
@@ -216,6 +218,23 @@ private async Task<TestResult> RunTestAsyncInternal(AssemblyInfo assemblyInfo, b
                     testResultInfo,
                     commandLineArguments,
                     processResults: ImmutableArray.CreateRange(processResultList));
+
+                void MaybeAddStressEnvironmentVariables()
+                {
+#if NETCOREAPP
```

Comment:
It just occurred to me that RunTests.dll itself always defines this symbol even if we are running net472 tests. May be more appropriate to check for the absence of `"net472"` in Options.TargetFrameworks

Reply:
Doh! You're right I missed that. I will get that fixed up in a bit.